modules/space/rendering/renderablefluxnodes.cpp

```diff
@@ -0,0 +1,1106 @@
+                columnValue.replace(16, 1, ":");
+                columnValue.replace(19, 1, ".");
+                const double triggerTime = Time::convertTime(columnValue);
+                LDEBUG("timestring " + columnValue);
+                _startTimes.push_back(triggerTime);
+            }
+            else {
+                LERROR(fmt::format("Error in file formating. Last column in ",
+                    "file '{}' is not on UTC ISO8601 format", timeFile
+                ));
+            }
+            
+        }
+    }
+}
+
+void RenderableFluxNodes::updateActiveTriggerTimeIndex(double currentTime) {
+    auto iter = std::upper_bound(_startTimes.begin(), _startTimes.end(), currentTime);
+    if (iter != _startTimes.end()) {
+        if (iter != _startTimes.begin()) {
+            _activeTriggerTimeIndex = static_cast<int>(
+                std::distance(_startTimes.begin(), iter)
+            ) - 1;
+        }
+        else {
+            _activeTriggerTimeIndex = 0;
+        }
+    }
+    else {
+        _activeTriggerTimeIndex = static_cast<int>(_nStates) - 1;
+    }
+}
+void RenderableFluxNodes::render(const RenderData& data, RendererTasks&) {
+    if (_activeTriggerTimeIndex != -1) {
+        _shaderProgram->activate();
+
+        // Calculate Model View MatrixProjection
+        const glm::dmat4 rotMat = glm::dmat4(data.modelTransform.rotation);
+        const glm::dmat4 modelMat =
+            glm::translate(glm::dmat4(1.0), data.modelTransform.translation) *
+            rotMat *
+            glm::dmat4(glm::scale(
+                glm::dmat4(1.0), glm::dvec3(data.modelTransform.scale)
+            ));
+        const glm::dmat4 modelViewMat = data.camera.combinedViewMatrix() * modelMat;
+
+        //not in use atm.
+        _shaderProgram->setUniform("modelViewProjection",
+            data.camera.sgctInternal.projectionMatrix() * glm::mat4(modelViewMat)
+        );
+
+        SceneGraphNode* earthNode = sceneGraphNode("Earth");
+        if (!earthNode) {
+            LWARNING("Could not find scene graph node 'Earth'.");
+        }
+        glm::vec3 earthPos = earthNode->worldPosition() * data.modelTransform.rotation;
+    
+        _shaderProgram->setUniform(_uniformCache.streamColor, _streamColor);
+        _shaderProgram->setUniform(_uniformCache.nodeSize, _nodeSize);
+        _shaderProgram->setUniform(
+            _uniformCache.nodeSizeLargerFlux, 
+            _nodeSizeLargerFlux
+        );
+        _shaderProgram->setUniform(_uniformCache.thresholdFlux, _thresholdFlux);
+        _shaderProgram->setUniform(_uniformCache.colorMode, _colorMode);
+        _shaderProgram->setUniform(_uniformCache.filterLower, _filteringLower);
+        _shaderProgram->setUniform(_uniformCache.filterUpper, _filteringUpper);
+        _shaderProgram->setUniform(_uniformCache.scalingMode, _scalingmethod);
+        _shaderProgram->setUniform(
+            _uniformCache.colorTableRange, 
+            _colorTableRange.value()
```

Comment:
I don't think the `.value()` function necessary.  If that is removed, you can compress these four lines together into one, too